evm/logic/call.py

```diff
@@ -105,14 +109,18 @@ def __call__(self, computation):
             if child_computation.error:
```

Comment:
I don't think so or I'm not understanding your suggestion.

Reply:
I meant `if child_computation.error is not None:`, just in case at some point we have an error class that for some reason implements `__bool__()` and returns False there, this would no longer work as expected. I can't think why we'd have an error class that would do that, but in general I prefer to explicitly check for None

Reply:
👍 I'm going to file this away in a "Good First Issue" ticket.